specs/eip4844/p2p-interface.md

```diff
@@ -0,0 +1,264 @@
+# EIP-4844 -- Networking
+
+This document contains the consensus-layer networking specification for EIP-4844.
+
+The specification of these changes continues in the same format as the network specifications of previous upgrades, and assumes them as pre-requisite.
+
+## Table of contents
+
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+  - [Preset](#preset)
+  - [Configuration](#configuration)
+  - [Containers](#containers)
+    - [`BlobsSidecar`](#blobssidecar)
+    - [`SignedBlobsSidecar`](#signedblobssidecar)
+  - [The gossip domain: gossipsub](#the-gossip-domain-gossipsub)
+    - [Topics and messages](#topics-and-messages)
+      - [Global topics](#global-topics)
+        - [`beacon_block`](#beacon_block)
+        - [`blobs_sidecar`](#blobs_sidecar)
+    - [Transitioning the gossip](#transitioning-the-gossip)
+  - [The Req/Resp domain](#the-reqresp-domain)
+    - [Messages](#messages)
+      - [BeaconBlocksByRange v2](#beaconblocksbyrange-v2)
+      - [BeaconBlocksByRoot v2](#beaconblocksbyroot-v2)
+      - [BlobsSidecarsByRange v1](#blobssidecarsbyrange-v1)
+- [Design decision rationale](#design-decision-rationale)
+  - [Why are blobs relayed as a sidecar, separate from beacon blocks?](#why-are-blobs-relayed-as-a-sidecar-separate-from-beacon-blocks)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+
+## Preset
+
+| Name | Value |
+| - | - |
+| `LIMIT_BLOBS_PER_SIDECAR` | `uint64(2**4)` (= 16) |
+
+## Configuration
+
+| Name                                     | Value                         | Description                                                         |
+|------------------------------------------|-------------------------------|---------------------------------------------------------------------|
+| `MAX_REQUEST_BLOBS_SIDECARS`             | `2**10` (= 1024)              | Maximum number of blobs sidecars in a single request                |
```

Comment:
This could be a huge amount of data (on order of 1GB)!

Is this okay to be made in a single request? This seems quite excessive to *not* be load balancing to some extent across peers